pkg/router/template/router.go

```diff
@@ -398,6 +399,12 @@ func (r *templateRouter) AddRoute(id string, route *routeapi.Route, host string)
 	config := ServiceAliasConfig{
 		Host: host,
 		Path: route.Spec.Path,
+		Hash: backendKey,
+	}
+
+	// Possibly hash the name for later use in obsuring cookie ID
+	if os.Getenv("COOKIE_PREFIX_MAIN") + os.Getenv("COOKIE_PREFIX_EDGE") + os.Getenv("COOKIE_PREFIX_REENCRYPT") != "" {
```

Comment:
Can we just do it with a flag (in lieu of 3 env vars) that does this equivalent of `"obfuscating"` the cookie name. Or maybe even just do it by default?  
Maybe hash it based on the `tls.Termination` + `backendKey` as it would have the type of route as well. Something like:
```  
    Hash:  fmt.Sprintf("%s_%s", tls.Termination, backendKey),  
  }  
// ...  
if  r.TheObsfucatorFlagName { // replace flag name w/ the real one.  
   config.Hash = fmt.Sprintf("%x", md5.Sum([]byte(config.Hash)))  
}  
```